bin/p2-watch/main.go

```diff
@@ -35,13 +34,13 @@ func main() {
 
 	quit := make(chan struct{})
 	errChan := make(chan error)
-	podCh := make(chan pods.PodManifest)
+	podCh := make(chan intent.ManifestResult)
```

Comment:
I switched this to a wrapper object so I could see what the path in Consul was when we read the pod.